src/BuildScriptGenerator/DotNetCore/DefaultAspNetCoreWebAppProjectFileProvider.cs

```diff
@@ -171,6 +169,25 @@ internal static bool IsAspNetCoreWebApplicationProject(XDocument projectFileDoc)
             return string.Equals(sdkName, expectedWebSdkName, StringComparison.OrdinalIgnoreCase);
         }
 
+        // To enable unit testing
+        internal static string GetRelativePathToRoot(string projectFilePath, string repoRoot)
+        {
+            var repoRootDir = new DirectoryInfo(repoRoot);
```

Comment:
how about getting the full path for both, and just getting the `repoRoot` substring starting at `projectFilePath.Length +1`?